src/sparsezoo/model/utils.py

```diff
@@ -54,6 +57,49 @@
 SAVE_DIR = os.getenv("SPARSEZOO_MODELS_PATH", CACHE_DIR)
 
 
+class ModelResult(BaseModel):
+    """
+    Base class to store common result information
```

Comment:
I personally would rather have the explanation of the schema's properties inside the `Field`.

Reply:
Added!